CloudyCider/models/EC2Store.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  EC2Store.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/7/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import EC2
+import AWSSDKSwiftCore
+
+final class EC2Store: BindableObject {
+  let didChange = PassthroughSubject<(), Never>()
+
+  var result: Result<[EC2Instance], Error> = .success([]) {
+    didSet {
+      DispatchQueue.main.async {
+        self.didChange.send()
+      }
+    }
+  }
+    
+    func loadPage() {
```

Comment:
the formatting here uses 4 spaces, which the rest of the code above uses 2 spaces